spec/swagger/swagger_doc_v1.yaml

```diff
@@ -550,6 +636,31 @@
           description: The move object to be modified
           schema:
             "$ref": "../v1/move.yaml#/Move"
+        - name: include
+          description: Returns a specific list of related resources to the move
+          in: query
+          style: form
+          explode: false
+          schema:
+            type: string
+            enum:
+              - allocation
+              - court_hearings
+              - documents
+              - from_location
+              - to_location
+              - original_move
+              - prison_transfer_reason
+              - person
+              - profile
+              - from_location.suppliers
+              - to_location.suppliers
+              - person.gender
+              - person.ethnicity
+              - profile.person
+              - profile.person.gender
+              - profile.person.ethnicity
```

Comment:
These are repeated above - might be worth extracting out into something like `move_include_attribute.yaml` to DRY this.